backend/models/GpsSettings.js

```
@@ -0,0 +1,191 @@
+      throw error;
+    }
+  }
+
+  // Obtenir les paramètres GPS d'un livreur
+  static async getByLivreurId(livreur_id) {
+    const query = 'SELECT * FROM gps_settings WHERE livreur_id = $1';
+    const result = await db.query(query, [livreur_id]);
+    
+    if (result.rows.length === 0) {
+      // Créer des paramètres par défaut si ils n'existent pas
+      return this.createOrUpdate({ 
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)